### PR TITLE
Add autoware version node

### PR DESCRIPTION
**Pull request by isamu-takagi**

### CODE REVIEW

File: system/autoware_version/src/autoware_version.cpp [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+#include <memory>
+#include <string>
+#include "rclcpp/rclcpp.hpp"
+#include "autoware_system_msgs/msg/autoware_version.hpp"
+
+class AutowareVersionNode : public rclcpp::Node
+{
+public:
+  AutowareVersionNode()
+  : Node("lane_change_planner_node")
```
Comment (kenji-miyake) [inline system/autoware_version/src/autoware_version.cpp:25]: old name remains.

Comment (isamu-takagi) [reply system/autoware_version/src/autoware_version.cpp]: fixed.

**Comment by mitsudome-r** [comment]
I've changed the target branch to ros2

**Review by mitsudome-r — APPROVED** [review]
LGTM

**Pull request merged**